.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: Feature Request
+description: Suggest an idea for improving RDF4J
+labels: [ "📶 enhancement", "triage" ]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to let us know your suggestion.
+    - type: textarea
+      attributes:
+        label: Problem description
+        description: Provide a concise description of the problem your improvement will address.
+        placeholder: "Performing task X would be a lot easier if RDF4J could do..." 
+      validations:
+        required: true
+    - type: textarea
+      attributes:
+        label: Preferred solution
+        description: Provide a clear and concise description of what you want to happen.
```

Comment:
Any way we could encourage code contribution. I want to keep expectations aligned with our dev capacity without making people feel that they can't ask for a feature while at the same time gently reminding them that we are a small open source project.

Reply:
Good point. I'm not sure if we want that in the placeholder text or perhaps more prominent.

Reply:
@hmottestad I've fixed this by actually ading a (non-required) extra field to ask if they're interested in contributing themselves.